.idea/codeStyleSettings.xml

```diff
@@ -1,264 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We probably want to keep the .idea files?

Reply:
Yes, Android Studio keeps doing that automatically somehow. Will revert it.